src/Models/Key.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Tipoff\LaravelSerpapi\Models;
+
+use Tipoff\Support\Models\BaseModel;
+use Tipoff\Support\Traits\HasPackageFactory;
+
+class Key extends BaseModel
```

Comment:
We have another model in the tipoff/laravel-google-api package with the name of Key. We should maybe use a different name here or use that same model and I can move it to tipoff/authorization